helmfile.d/traefik.yaml

```diff
@@ -0,0 +1,20 @@
+repositories:
+  - name: traefik
+    url: 
+releases:
+- name: traefik
+  chart: stable/traefik
+  namespace: kube-system
+  version: 1.3.0
+  wait: true
+  timeout: 300
+  atomic: true
+values:
+  - defaultBackend:
+      enabled: true
+      image:
+        repository: jenkinsciinfra/404
```

Comment:
This is just to replace the default 404 with something more jenkensie?

Reply:
Yes it's just a Jenkins 404 page :-p